portfolio/src/main/webapp/script.js

```diff
@@ -160,17 +165,71 @@ function deleteComments() {
   fetch(request).then(response => getComments());
 }
 
+/** Fetches login and Blobstore. */
```

Comment:
Fetches login what? Make sure it's clear exactly what's being fetched!

Reply:
Done!